MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1886,12 +1886,9 @@ public void writeToXml(PrintWriter pw1, int indent) {
     public static Person generateInstanceFromXML(Node wn, Campaign c, Version version) {
         final String METHOD_NAME = "generateInstanceFromXML(Node,Campaign,Version)"; //$NON-NLS-1$
 
-        Person retVal = null;
+        Person retVal = new Person(c);
```

Comment:
Do we really want a non-null person in case we fail to read a person in?

Reply:
We already will have a non-null person, as the only way it will return null currently is if it fails to create the person object initially.

However, by adding retVal = null on the catch would do that, which seems to have been the original intent in the code. Pushed a fix to ensure the original intent works.